webpack.config.js

```diff
@@ -104,6 +106,7 @@ module.exports = {
     ]),
     new ExtractTextPlugin(`[name]${hash}.css`),
     new ManifestPlugin(),
+    new GenerateJsonPlugin('../src/js/languages.json', generatedLangList),
```

Comment:
In case you want to prettify the output.

```javascript
GenerateJsonPlugin(<file>, <json>, [replacer], [space])
```

Fill out that `space` argument.

Reply:
Ok